tests/framework/installer/ceph_manifests.go

```diff
@@ -1200,7 +1206,7 @@ rules:
     verbs: ["list", "watch", "create", "update", "patch"]
   - apiGroups: ["snapshot.storage.k8s.io"]
     resources: ["volumesnapshots"]
-    verbs: ["get", "list", "watch"]
+    verbs: ["get", "list", "watch", "patch"]
```

Comment:
Are all these RBAC changes already in the common.yaml and helm chart?

Reply:
Yes, these changes are already present in common.yaml and helm chart